src/redux/actions.js

```diff
@@ -88,12 +88,6 @@ export function mapEquipmentsCreated(
     };
 }
 
-export const LOAD_GEO_DATA_SUCCESS = 'LOAD_GEO_DATA_SUCCESS';
```

Comment:
used anywhere ? Ok to clean this

Reply:
Yes it's some old code. It's not used anymore for a while